src/sentry/utils/sentryappwebhookrequests.py

```diff
@@ -0,0 +1,136 @@
+from __future__ import absolute_import
+
+import six
+import json
+
+from dateutil.parser import parse as parse_date
+from django.conf import settings
+from django.utils import timezone
+
+from sentry.utils import redis
+from sentry.models.sentryapp import VALID_EVENTS
+
+
+BUFFER_SIZE = 100
+KEY_EXPIRY = 60 * 60 * 24 * 30  # 30 days
+
+
+class SentryAppWebhookRequestsBuffer(object):
+    """
+    Create a data structure to store basic information about Sentry App webhook requests in Redis
+    This should store the last 100 requests and last 100 errors (in different keys) for each event type, for each Sentry App
+    """
+
+    def __init__(self, sentry_app):
+        self.sentry_app = sentry_app
+
+        cluster_id = getattr(settings, "SENTRY_WEBHOOK_LOG_REDIS_CLUSTER", "default")
+        self.client = redis.redis_clusters.get(cluster_id)
```

Comment:
I don't think any of the `if self.client is not None` branches are going to be hit below, `ClusterManager` is going to throw if it can't return an actual cluster: https://github.com/getsentry/sentry/blob/370172efe9cd769ec5b9a24cb3682b25bc1b6859/src/sentry/utils/redis.py#L147-L152

Reply:
@tkaemming oops yeah you're right, forgot to remove them